src/scala/microsoft-spark-2.4.x/src/main/scala/org/apache/spark/sql/api/dotnet/SQLUtils.scala

```diff
@@ -10,7 +10,6 @@ import java.util.{List => JList, Map => JMap}
 
 import org.apache.spark.api.python.{PythonAccumulatorV2, PythonBroadcast, PythonFunction}
 import org.apache.spark.broadcast.Broadcast
-import org.apache.spark.deploy.dotnet.DotnetRunner
```

Comment:
3.0 is fine?

Reply:
huh. I thought I removed this `createBroadcast` function after using `javaSparkContext`, in fact I do not see it locally, probably made some mistake while pushing. Removing this as it's not needed.

Reply:
Nope, good catch. Removed now. Thanks!